_test/tests/lib/exe/css_at_import_less.test.php

```
@@ -14,17 +14,17 @@ public function setUpFiles($subdir = '') {
             mkdir($dir, 0777, true);
         }
         if (!is_dir($dir)) {
-            $this->markTestSkipped('Could not create directory.');
+            throw new Exception('Could not create directory.');
```

throw exception instead of skip because this is issue